src/test/scala/uk/gov/nationalarchives/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -0,0 +1,126 @@
+package uk.gov.nationalarchives.testUtils
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import fs2.interop.reactivestreams.StreamUnicastPublisher
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers.any
+import org.mockito.MockitoSugar.{mock, times, verify, when}
+import org.reactivestreams.Publisher
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers._
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import software.amazon.awssdk.core.async.SdkPublisher
+import software.amazon.awssdk.services.s3.S3AsyncClient
+import software.amazon.awssdk.services.s3.model.{CommonPrefix, ListObjectsV2Request, ListObjectsV2Response}
+import software.amazon.awssdk.services.s3.paginators.ListObjectsV2Publisher
+import software.amazon.awssdk.transfer.s3.model.CompletedUpload
+import uk.gov.nationalarchives.{DAS3Client, Lambda}
+
+import java.nio.ByteBuffer
+import java.util.concurrent.CompletableFuture
+import scala.jdk.CollectionConverters.SeqHasAsJava
+
+class ExternalServicesTestUtils
+    extends AnyFlatSpec
+    with BeforeAndAfterEach
```

Comment:
Do you need the BeforeAndAfterEach, BeforeAndAfterAll and TableDrivenPropertyChecks traits? It doesn't look like you're using them.